content/learn/learn.md

```diff
@@ -12,8 +12,7 @@ an [NHGRI](https://www.genome.gov/Funded-Programs-Projects/Computational-Genomic
 and controlled access genomic [datasets](https://anvilproject.org/data/consortia) with popular analysis tools in
 a [secure](/overview/security) cloud computing environment.
 
-AnVIL uses [Terra](https://anvil.terra.bio/#workspaces) as its analysis platform, [Gen3](https://gen3.theanvil.io) for
```

Comment:
Thanks, @avahoffman, for this PR! Can you reference https://explore.anvilproject.org/ as the data search and artificial cohort creation platform? Cheers, D

Reply:
you got it. Change incoming!